packages/gatsby/gatsby-browser.js

```diff
@@ -1,15 +1,15 @@
 const Sentry = require('@sentry/react');
 const Tracing = require('@sentry/tracing');
+const Utils = require('@sentry/utils')
 
 exports.onClientEntry = function(_, pluginParams) {
   if (pluginParams === undefined) {
     return;
   }
 
-  const tracesSampleRate = pluginParams.tracesSampleRate !== undefined ? pluginParams.tracesSampleRate : 0;
   const integrations = [...(pluginParams.integrations || [])];
 
-  if (tracesSampleRate && !integrations.some(ele => ele.name === 'BrowserTracing')) {
+  if (Utils.hasTracingEnabled(pluginParams) && !integrations.some(ele => ele.name === 'BrowserTracing')) {
```

Comment:
Perhaps `hasTracingEnabled` would be best suited to live in `@sentry/tracing`. As we can see here, `@sentry/gatsby` did not depend on `@sentry/utils` and now it does just because of... tracing. We shouldn't need to pull in all of utils for a single tracing-related helper.

Reply:
Fair point. Done.